gcc/rust/resolve/rust-toplevel-name-resolver-2.0.h

```diff
@@ -59,8 +82,10 @@ class TopLevel : public DefaultResolver
 
   // FIXME: Do we move these to our mappings?
   std::unordered_map<NodeId, location_t> node_locations;
+  std::unordered_map<NodeId, NodeId> node_forwarding;
```

Comment:
an you add a small comment explaining the point of this map?

Reply:
Fixed.